src-java/kilda-persistence-tinkerpop/src/test/java/org/openkilda/persistence/ferma/repositories/FermaHaSubFlowRepositoryTest.java

```diff
@@ -145,8 +152,8 @@ private void assertSubFlow(
     }
 
     private HaSubFlow createSubFlow(
-            String subFlowId, SwitchId switchId, int port, int vlan, int innerVlan, String description) {
-        HaSubFlow subFlow = buildHaSubFlow(subFlowId, switchId, port, vlan, innerVlan, description);
+            String subFlowId, Switch sw, int port, int vlan, int innerVlan, String description) {
```

Comment:
no need to shorten, use switch instead

Reply:
`switch` is reserved word in Java. We can't use it